client/core/core.go

```diff
@@ -4037,6 +4041,45 @@ func handleNoMatchRoute(c *Core, dc *dexConnection, msg *msgjson.Message) error
 	return err
 }
 
+func (c *Core) schedTradeTick(tracker *trackedTrade) {
+	oid := tracker.ID()
+	c.tickSchedMtx.Lock()
+	defer c.tickSchedMtx.Unlock()
+	if _, found := c.tickSched[oid]; found {
+		return // already going to tick this trade
+	}
+
+	tick := func() {
+		assets, err := c.tick(tracker)
+		if len(assets) > 0 {
+			tracker.dc.refreshMarkets()
+			c.updateBalances(assets)
+		}
+		if err != nil {
+			c.log.Errorf("tick error for order %v: %v", oid, err)
+		}
+	}
+
+	numMatches := len(tracker.activeMatches())
+	if numMatches == 1 {
+		go tick()
+		return
+	}
+
+	// Schedule a tick for this trade.
+	delay := time.Duration(numMatches) * time.Second / 10 // 1 sec delay for every 10 active matches
+	if delay > 5*time.Second {
+		delay = 5 * time.Second
+	}
```

Comment:
Thinking we can infer an appropriate delay based on the number of active matches, not that all the active matches are in the same state, but it's at least an indicator of the upper limit on the reasonable delay.

Reply:
What we're actually up against is the various latencies of all counterparties. I like the idea of increasing the delay a little bit when there are a lot of matches, but I'm thinking 200 ms is too low for 2 matches. How about a constant term like

`delay := 2 * time.Second + time.Duration(numMatches) * time.Second / 10`

Reply:
That's what I realized reviewing logs, there's other party latency here, especially when there are multiple maker accounts for the different matched makers.  Gonna try a constant lower limit for that as you suggested.